docs/getting-started-guides/aws.md

```diff
@@ -2,12 +2,31 @@
 assignees:
 - justinsb
 - lavalamp
+- clove
 
 ---
 
 * TOC
 {:toc}
 
+
+## Supported Production Grade Tools with High Availability Options
+
+* [Kubernetes Operations](https://github.com/kubernetes/kops) - Production Grade K8s Installation, Upgrades, and Management. Supports running Debian and CentOS in AWS.
+
+* CoreOS maintains [a CLI tool](https://coreos.com/kubernetes/docs/latest/kubernetes-on-aws.html), `kube-aws` that will create and manage a Kubernetes cluster based on [CoreOS](http://www.coreos.com), using AWS tools: EC2, CloudFormation and Autoscaling.
+
+## Other Options
+
+* Other community projects exist that use tools such that use other configuration management tooling.
+
+TODO: add more options
+
+## Legacy Tooling
+
+`kube-up.sh` is a legacy tool that is an easy way to spin up a cluster.  This tool
+is being deprecated, and does not create a production ready environment.
+
 ## Prerequisites
```

Comment:
Do we want this in here?  Maybe we should move it elsewhere, or at least put it at `###` so that it nests logically under legacy tooling (though I expect this will not be obvious even then)